### DISCUSSION TITLE
Feature Request - GitHub Project Boards - Support for Dynamically Calculated Fields

### DISCUSSION THREAD

**Discussion by gauthierpetetin-test** [community/community#55839] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

Hello GitHub Team,

I'm writing to propose a feature that could enhance the functionality of GitHub project boards: dynamically calculated fields.

In my projects, I track four parameters for each task: `#Reach`, `#Impact`, `#Confidence`, and `#Effort`. These fields are currently static, allowing only fixed numeric values. However, the ability to have them work similarly to cells in an Excel sheet, where they can contain a formula and calculate a value dynamically, would be extremely beneficial.

For instance, I'd like to create a "RICE Score" field that calculates its value based on the existing fields with the following formula:

    RICE score = (#Reach * #Impact * #Confidence) / #Effort

This feature would enable real-time updates of key metrics and significantly streamline task prioritisation, providing benefits to many GitHub project board users.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by rileybroughten** [comment, marked as answer]
👋 Thank you for your feedback! I'm going to close this out in favor of https://github.com/orgs/community/discussions/6080 to keep the discussion and updates in a single place, but we hear you and are working to add additional fields and field types that give you more flexibility 👍

**Reply by yessiros** [reply]
hello riley

my name yesser henia and im from tunisia

My Sincere greetings to you and thank you for giving me the opportunity to written. I have a passion for the world of cars since childhood, especially the aesthetic side of the car. Therefore, I am writing this letter to you, to ask you if you can help me to submit a proposal to GitHub, which is a request for cooperation and partnership to implement the idea of a unique platform in terms of concept and  i will be grateful for you.
This concept is written on 3 PDF files clearly detailed that specializes in a specific field of the automotive world, which is very popular with people, especially the youth group, as it combines (entertainment and e-commerce) in terms of classification.
 Unfortunately, after registering the concept of platform in WIPO, the "World Intellectual Property Organization", I faced a major obstacle, which is providing financial capabilities and cooperation with a development company that has a global reputation and an important factor in terms of experience and competence that allows it to implement it on the ground and present it at the global level that I aspire to for her.

finally i want thank you very much for your attention and i wish have an answer from you.
